### DISCUSSION TITLE
Chatwoot  (live-chat, email support, omni-channel desk)

### DISCUSSION THREAD

**Discussion by tekos-flow** [coollabsio/coolify#2252] [Service Template Requests]
https://github.com/chatwoot/chatwoot

**Comment by Flowko** [comment]
god, this will be a game changer

**Comment by w7tf** [comment]
![image](https://github.com/coollabsio/coolify/assets/53350046/abeff591-fcfa-423e-b686-b2dd50fe44ec)


Seems like it has been added :)

**Comment by andrasbacsai** [comment]
> https://github.com/chatwoot/chatwoot

It is already added.

**Reply by gvzq** [reply]
> > https://github.com/chatwoot/chatwoot
> 
> It is already added.

Thank you for adding this service to this amazing project.

**Comment by gvzq** [comment]
There seems to be an issue in the default docker compose service stack.

In the Rails service: 
```
2024-07-19T16:40:49.875718372Z exec docker/entrypoints/rails.sh: exec format error
```

In the Sidekiq service:
```
2024-07-19T16:41:43.849492428Z exec /usr/local/bundle/bin/bundle: exec format error
```
